xla/service/llvm_ir/llvm_util.cc

```diff
@@ -377,8 +377,9 @@ llvm::AllocaInst* EmitAllocaAtFunctionEntryWithCount(llvm::Type* type,
   llvm::Function* function = b->GetInsertBlock()->getParent();
   b->SetInsertPoint(&function->getEntryBlock(),
                     function->getEntryBlock().getFirstInsertionPt());
+  // Some backends requires explicit local addrspace.
```

Comment:
By adding a comment, I meant something like this. (Please also apply a code formatter.)
```suggestion
  // Some backends requires explicit local addrspace. addrspace 5 maps to Intel accelerators.
  // Source: https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/docs/SPIRVRepresentationInLLVM.rst#address-spaces
```